File: Backend/UserService/src/main/java/com/user/dto/LeaderBoardOnSixMonthRewardPoints.java
package com.user.dto;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@NoArgsConstructor
@AllArgsConstructor
@Data
public class LeaderBoardOnSixMonthRewardPoints {
	private String name;
	private String city;
	private Double sixMonthRewardPoints;
	private boolean isCurrentUser = false;
	
	public LeaderBoardOnSixMonthRewardPoints(String name, String city, Double sixMonthRewardPoints) {
		super();
		this.name = name;
		this.city = city;
		this.sixMonthRewardPoints = sixMonthRewardPoints;
	}
	
	
}

File: FrontEnd/Self-Tracker/src/app/models/leader-board-on-six-month-reward-points.ts
export class LeaderBoardOnSixMonthRewardPoints {
  name: string;
  city: string;
  sixMonthRewardPoints: number;
  isCurrentUser: boolean;

  constructor(name: string, city: string, sixMonthRewardPoints: number, isCurrentUser: boolean) {
    this.name = name;
    this.city = city;
    this.sixMonthRewardPoints = sixMonthRewardPoints;
    this.isCurrentUser = isCurrentUser;
  }
}
